kontrakter/da/varetekt/arbeidsversjon/KjennelseVaretekt.schema.json

```diff
@@ -209,19 +210,80 @@
       "type": "object",
       "additionalProperties": false
     },
-    "anke": {
+    "erklaering": {
+      "properties": {
+        "erklaeringSiktede": {
+          "$ref": "#/definitions/erklaeringsDetaljer"
+        },
+        "erklaeringPaatale": {
+          "$ref": "#/definitions/erklaeringsDetaljer"
+        }
+      },
+      "required": ["erklaeringSiktede", "erklaeringPaatale"],
+      "type": "object",
+      "additionalProperties": false
+    },
+    "erklaeringsDetaljer": {
+      "properties": {
+        "meddeltIRettsmoete": {
+          "type": "boolean",
+          "description": "Settes til true desom avgjørelsen er meddelt i rettsmøtet"
+        },
+        "erklaert": {
+          "$ref": "#/definitions/erklaertType",
+          "description": "Settes dersom avgjørelsen er meddelt i retten"
+        },
+        "ankeDetaljer": {
+          "$ref": "#/definitions/ankeDetaljer",
+          "description": "Settes dersom parten har anket hele eller deler av avgjørelsen"
+        }
+      },
+      "required": ["meddeltIRettsmoete"],
+      "type": "object",
+      "additionalProperties": false
+    },
+    "ankeDetaljer": {
       "properties": {
         "anketDato": {
           "$ref": "#/definitions/dato"
         },
         "oppsettendeVirkning": {
-          "type": "boolean"
+          "type": "boolean",
+          "description": "Settes kun dersom det er påtale som anker"
+        },
+        "ankedeForhold": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/anketForholdType"
+          },
+          "minItems": 1,
```

Comment:
Vi håndhever ikke dette kravet i DA-frontend i dag og har ikke fått ønske om det fra design/test/fyrtårn.
Hvis denne skal stå som `1` må vi endre på vår side for å sikre at alle kjennelser validerer.

Reply:
Skjønner. Endrer denne til 0 da det ikke er påkrevd at noen forhold velges.